src/back/spv/writer.rs

```diff
@@ -1,12 +1,16 @@
+mod index;
```

Comment:
can we make it just `back/spv/index.rs` instead of `back/spv/writer/index.rs`?

Reply:
I've implemented "approach C", and it was fine.